cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -366,7 +367,10 @@ func GetAvailableResources(clientBuilder controller.ControllerClientBuilder) (ma
 
 	resourceMap, err := discoveryClient.ServerResources()
 	if err != nil {
-		return nil, fmt.Errorf("failed to get supported resources from server: %v", err)
+		utilruntime.HandleError(fmt.Errorf("unable to get all supported resources from server: %v", err))
```

Comment:
> HandleError only logs here?

The choice is up to the admin I suppose, there's an env var to force it either way.  Returning an error unconditionally fails, so this at least gives them the choice.



> Does this fix the issue where if I have two versions of a TPR, the cm crash loops?

I'm not familiar with this.  Got an issue handy?